tests/apt/test_install.py

```diff
@@ -0,0 +1,93 @@
+import itertools
+import pathlib
+import tempfile
+import typing
+import unittest.mock
+
+import pytest
+import pytest_console_scripts
+import typeguard
+
+from system_helpers.apt import install
+
+@pytest.fixture
+@typeguard.typechecked
+def requirements() -> typing.Generator[typing.Tuple[typing.List[pathlib.Path], typing.List[str]], None, None]:
+    """
+    Get requirements files *à la* `pip`.
+    """
+    with tempfile.NamedTemporaryFile(mode = 'w+') as req_1, \
+         tempfile.NamedTemporaryFile(mode = 'w+') as req_2:
+        req_1.write("# Let's start this first requirement file with a comment. Then, add some packages.\n")
+        req_1.write("git\n")
+        req_1.write("and\n")
+        req_1.write("\n")
+        req_1.write("# Some other useless comments here.\n")
+        req_1.write("whatnot\n")
+        req_1.flush()
+
+        req_2.write("# Let's start this other requirement file with a comment. Then, add some packages.\n")
+        req_2.write("cmake\n")
+        req_2.write("is\n")
+        req_2.write("nice even with many whatever\n")
+        req_2.flush()
+
+        packages = ['git', 'and', 'whatnot', 'cmake', 'is', 'nice', 'even', 'with', 'many', 'whatever']
+
+        yield ([pathlib.Path(req_1.name), pathlib.Path(req_2.name)], packages)
+
+class TestAptInstall:
+    """
+    Test :py:class:`apt_helpers.install.apt_install_packages`.
+    """
+    @staticmethod
+    @typeguard.typechecked
+    def get_script():
+        """
+        Retrieve script path.
+        """
+        return pathlib.Path(__file__).parent.parent.parent / 'system_helpers' / 'apt' / 'script.py'
+
+    def test_list_of_package_names(self):
+        """
+        Test for a provided list of package names.
+        """
+        packages = ['git', 'and', 'whatnot']
+
+        with unittest.mock.patch(target = 'subprocess.check_call', side_effect = [None, None, None]) as mocker:
+            install.install_packages(packages = packages, update = False, upgrade = True, clean = False)
+
+            mocker.assert_has_calls(calls = [
+                unittest.mock.call(['apt', '--yes', 'upgrade']),
+                unittest.mock.call(['apt', '--yes', '--no-install-recommends', 'install'] + packages),
+            ])
+
+    def test_list_of_requirement_files(self, requirements):
```

Comment:
```suggestion
    def test_list_of_requirements_files(self, requirements):
```
You do a search replace? :)

Reply:
:heavy_check_mark: